### DISCUSSION TITLE
PackIcon "Transfer" disappeared?

### DISCUSSION THREAD

**Discussion by DrCQ** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2940] [Q&A] [answered]
I'm porting NET Framework to NET Core 6.0 and use MaterialDesignXamlToolkit Version 4.6.1/2.0.9 instead of 3.1.0/1.2.3. 
I cannot find the PackIcon "Transfer" anymore. What happens here (renamed, removed)?

**Comment by Keboo** [comment]
So, the PackIcons are a special case because they are automatically updated. When they are changed by 
https://materialdesignicons.com/ this library automatically ingests the changes. 

I suspect you are likely looking for the `TransferRight` or `TransferLeft` renames that occured.

**Comment by DrCQ** [comment]
Hi Kevin,

 

I’m looking for an icon which was called “Transfer” and looks like here: . It shows moving items from one stack to another. I cannot find it anymore.

 

Regards,

 

Christoph

 

Von: Kevin B ***@***.***> 
Gesendet: Sonntag, 27. November 2022 08:57
An: MaterialDesignInXAML/MaterialDesignInXamlToolkit ***@***.***>
Cc: Dr.CQ ***@***.***>; Author ***@***.***>
Betreff: Re: [MaterialDesignInXAML/MaterialDesignInXamlToolkit] PackIcon "Transfer" disappeared? (Discussion #2940)

 

So, the PackIcons are a special case because they are automatically updated. When they are changed by
https://materialdesignicons.com/ this library automatically ingests the changes.

I suspect you are likely looking for the TransferRight or TransferLeft renames that occured.

**Comment by Keboo** [comment, marked as answer]
I might be missing something but why I search in the demo app. I do appear to find that icon.
![image](https://user-images.githubusercontent.com/952248/204153701-5cf27e46-ece2-4ca6-a856-c5e3cfb44a6c.png)

**Comment by DrCQ** [comment]
It’s magic.

 

While using Blend I got an error that this “kind” doesn’t exist. It was really surprising me and I posted my question. At the same time I checked the online version on matererialdesignicons.com and also miss this icon.

 

Now I checked the demo app and the online version – this icon is there.

 

Sorry for confusion, but I still don’t understand what happens (do we have new version since my post?)

 

Best regards,

 

Christoph

 

Von: Kevin B ***@***.***> 
Gesendet: Sonntag, 27. November 2022 19:43
An: MaterialDesignInXAML/MaterialDesignInXamlToolkit ***@***.***>
Cc: Dr.CQ ***@***.***>; Author ***@***.***>
Betreff: Re: [MaterialDesignInXAML/MaterialDesignInXamlToolkit] PackIcon "Transfer" disappeared? (Discussion #2940)

 

I might be missing something but why I search in the demo app. I do appear to find that icon.
 <https://user-images.githubusercontent.com/952248/204153701-5cf27e46-ece2-4ca6-a856-c5e3cfb44a6c.png>
